src/shared/apis/records/getStatistics.ts

```diff
@@ -0,0 +1,16 @@
+import { createQuery } from 'react-query-kit';
+
+import { Statistics } from '@/shared/types/record/statistics';
+import { request } from '@/shared/utils/request';
+
+const getStatistics = () => {
+  return request<Statistics>({
+    method: 'get',
+    url: 'records/me/statistics',
```

Comment:
url 형태는 `/~`로 시작하도록 통일되면 좋을 것 같습니다
```suggestion
    url: '/records/me/statistics',
```